index.html

```diff
@@ -606,13 +649,70 @@ <h3>The <dfn>observe()</dfn> method</h3>
           Run the following steps [=in parallel=]:
           <ol>
             <li>
-              If |source:PressureSource| is not a [=valid source type=],
-              [=queue a global task=] on the [=PressureObserver task source=]
-              given |relevantGlobal|
-              to reject |promise| {{NotSupportedError}} and abort these steps.
+              If |relevantGlobal|'s [=platform collector mapping=] does not
+              [=map/contain=] |source|:
+              <ol>
+                <li>
+                  Let |newCollector| be a new [=platform collector=] whose
+                  [=platform collector/associated pressure source=] is null.
+                </li>
+                <li>
+                  Let |virtualPressureSource| be the result of invoking [=get a
```

Comment:
should virtualPressureSource have a : type?

Reply:
Done in commit a58799e20bbe8f900bc3ca9a0bec5d740d60757c.